website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -166,17 +166,17 @@ These are the buckets that will be created in cumulus OU account:
 - `<PREFIX>-protected`
 - `<PREFIX>-tf-state` (for storing the terraform state file in cumulus OU account)
 
-After hitting the play button on `Deploy Dev RDS Stack`, but before hitting `Run` in the popup, replace the following variables with yours.
+While running the `Deploy ORCA Buckets and Cumulus/ORCA modules` stage, replace the following variables with yours. This is because some variables are sensitive and some will vary depending upon the user running the pipeline. Hitting 'play' next to `Deploy DR ORCA buckets`, `Deploy ORCA Buckets and Cumulus/ORCA modules` brings up a checkbox list to run multiple jobs at once. Note that none of the checkboxes should be checked.

```

Comment:
This was an intentional change on my part. This does not occur "while running", but before it is run.

Reply:
I concur

Reply:
fixed